.github/workflows/publish-oss.yml

```diff
@@ -25,6 +24,10 @@ on:
                 description: "Choose the branch to publish from"
                 type: string
                 default: "main"
+            version:
+                description: "Does the version need to be bumped?"
+                type: string
+                default: ""
```

Comment:
This reads as a yes/no question but it is looking for how to bump the version, right?